app/helpers/representatives_helper.rb

```diff
@@ -0,0 +1,13 @@
+module RepresentativesHelper
```

Comment:
Massa. Vamos trocar umas ideias sobre como vai ser o refatoramento quando você for fazer, beleza?